google/cloud/forseti/services/inventory/storage.py

```diff
@@ -521,12 +522,12 @@ def initialize(engine):
 class Storage(BaseStorage):
     """Inventory storage used during creation."""
 
-    def __init__(self, session, existing_id=None, readonly=False):
+    def __init__(self, session, existing_id=0, readonly=False):
         """Initialize
 
         Args:
             session (object): db session
-            existing_id (str): The inventory id if wants to open an existing one
+            existing_id (int64): The inventory id if wants to open an existing one
```

Comment:
Putting comment here for lack of a better place:

This should probably not return a string any longer either?

https://github.com/GoogleCloudPlatform/forseti-security/blob/1a8a62cd7f7fbaf4ac89d4a7c1349cf2c722d1d0/google/cloud/forseti/services/inventory/storage.py#L490

Reply:
This method is used in notifier, I have updated notifier to use inventory_index_id as int, too. PTAL again!